bouncer/shared/fund_redeem.ts

```diff
@@ -13,22 +13,21 @@ import { fundFlip } from '../shared/fund_flip';
 import { redeemFlip, RedeemAmount } from '../shared/redeem_flip';
 import { newStatechainAddress } from '../shared/new_statechain_address';
 
-// Submitting the `redeem` extrinsic will cost a small amount of gas
-// TODO: Send the redeem extrinsic from a different account to avoid compensating for this fee in the test.
-const expectedRedeemGasFeeFlip = 0.0000125;
+// Submitting the `redeem` extrinsic will cost a small amount of gas. Any more than this and we should be suspicious.
+const gasErrorMargin = 0.1;
 
 /// Redeems the flip and observed the balance increase
 async function redeemAndObserve(
   seed: string,
   redeemEthAddress: HexString,
   redeemAmount: RedeemAmount,
 ): Promise<number> {
-  const initBalance = await getBalance('Flip', redeemEthAddress);
-  console.log(`Initial ERC20-Flip balance: ${initBalance}`);
+  const initBalance = await getBalance('FLIP', redeemEthAddress);
+  console.log(`Initial ERC20-FLIP balance: ${initBalance}`);
```

Comment:
👀  doesn't really seem relevant to the change? Bad merge?

Reply:
Yeah, fixing.